includes/base/widget-base.php

```diff
@@ -647,7 +647,7 @@ public function render_content() {
 				$this->register_runtime_widget( $this->get_group_name() );
 			}
 
-			$this->print_widget_css();
+			// $this->print_widget_css();
```

Comment:
For more information about this change, ask @KingYes.

This change also requires removal of a test that is no longer relevant.

Reply:
Is it possible to remove it, instead of commenting it out?